components/browser/engine-system/src/main/java/mozilla/components/browser/engine/system/SystemEngineView.kt

```diff
@@ -248,6 +249,7 @@ class SystemEngineView @JvmOverloads constructor(
                     interceptor.onLoadRequest(
                         session,
                         request.url.toString(),
+                        lastOnLoadRequestUri,
```

Comment:
For engine-system, we can just use `session.currentUrl`, right? We don't have the same problem as in engine-gecko.

Reply:
I see.  I'll double check.   Thanks,

Reply:
Agreed, I'll fix.  Thanks,